src/internal/operations/backup_test.go

```diff
@@ -273,16 +273,22 @@ func makePath(t *testing.T, elements []string, isItem bool) path.Path {
 func makeDetailsEntry(
 	t *testing.T,
 	p path.Path,
+	l path.Path,
 	size int,
 	updated bool,
 ) *details.DetailsEntry {
 	t.Helper()
 
+	var lr string
+	if l != nil {
+		lr = l.PopFront().PopFront().PopFront().PopFront().Dir().String()
+	}
+
 	res := &details.DetailsEntry{
 		RepoRef:     p.String(),
 		ShortRef:    p.ShortRef(),
 		ParentRef:   p.ToBuilder().Dir().ShortRef(),
-		LocationRef: p.PopFront().PopFront().PopFront().PopFront().Dir().String(),
+		LocationRef: lr,
```

Comment:
ahhh now that this is a param I'm super tempted to ask for them not to be the same values in the tests so we know the code isn't conflating the two (or ignoring one of them). This kind of is along the lines of "what assumptions does my test make about the ways the code could possibly be broken?" But anyway, not a blocker for this to merge

Reply:
eh, might as well

Reply:
totally worth it: caught a bug